vmdb/app/models/configured_system_foreman/placement.rb

```diff
@@ -0,0 +1,14 @@
+module ConfiguredSystemForeman::Placement
+  def available_configuration_profiles
+    return [] if configuration_location.nil? || configuration_organization.nil?
+    cl_path = configuration_location.path
+    co_path = configuration_organization.path
+    ConfigurationProfile.joins(:configuration_locations, :configuration_organizations)
+      .includes(:configuration_architecture)
+      .where(
+        :configuration_locations_configuration_profiles     => {:configuration_location_id     => cl_path.collect(&:id)},
+        :configuration_organizations_configuration_profiles => {:configuration_organization_id => co_path.collect(&:id)},
```

Comment:
Can you move the `.collect(&:id)` up to the `.path` calls above?  We don't use `cl_path` for anything else, so if we move it up and change the variable name to `cl_path_ids`, then this gets a little less messy.